nexus/src/db/lookup.rs

```diff
@@ -527,6 +532,15 @@ lookup_resource! {
     ]
 }
 
+lookup_resource! {
+    name = "Rack",
+    ancestors = [],
+    children = [],
```

Comment:
Does a `Rack` not have `Sled`s as children?

Reply:
It could, but that relationship does not yet exist (`Sled`s do not have the FK to `Rack`s).

(Agreed that they should, eventually!)